docs/seqcol.md

```diff
@@ -442,7 +445,58 @@ The output of the comparison function provides information-rich feedback about t
 These details can be used to make a variety of inferences comparing two collections, but it can take some thought to interpret.
 For more details about how to interpret the results of the comparison function to determine different types of compatibility, please see the [howto guide on comparing sequencing collections](compare_collections.md).
 
-### 3.4 OpenAPI documentation
+
+#### 3.4 List
+
+- *Endpoint*: `GET /list/:object_type?page=:page&page_size=:page_size` (`REQUIRED`)
+- *Description*: Lists identifiers for a given object type (*e.g.* collections). This endpoint provides a way to discover what sequence collections a servic provides.
+- *Return value*: The output is a paged list of identifiers following the GA4GH paging guide format, grouped into a `results` and a `pagination` section.
+
+Example return value:
+
+```
+{
+  "results": [
+    ...
+  ],
+  "pagination": {
+    "page": 1,
+    "page_size": 100,
+    "total": 165,
+  }
+}
+```
+
+
+##### Variant: List with filter by attribute value
+
+The top-level `/list` endpoint will return all items (paged). A variant of this endpoint allows users to retrieve only certain items, filtered by attribute digest. Adding the `:attribute` and `:attribute_digest` to the endpoint accomplishes this:
+
+- *Endpoint*: `GET /list/:object_type/:attribute/:attribute_digest?page=:page&page_size=:page_size` (`REQUIRED`)
+- *Description*: Lists identifiers for a given object type (*e.g.* collections), filtered to only those that have a specific attribute value. This endpoint provides a way to discover sequence collections with a certain attribute.
```

Comment:
Be specific on the allowed value of object_type. Should it be `collection` or `collections`? Is it ok if the plurality is different than in the `attribute` and `collection` endpoints?

Reply:
Same for the description of the main `list` variant

Reply:
fixed, thx.